Cargo.toml

```diff
@@ -6,10 +6,17 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["windows"]
 windows = ["skia-safe/gl", "skia-safe/d3d"]
+macos = ["skia-safe/gl", "skia-safe/metal"]
 
 [dependencies]
 skia-safe = "0.53.0"
 glutin = "0.29"
 gl = "0.14.0"
+cocoa = "0.24.0"
```

Comment:
double check we need all of these dependencies or if some of them are dev dependencies